en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -378,11 +385,14 @@ To create one or more new variables using the *JSON Schema* window:
 . In the *JSON Schema* window, enter the properties for the variable to create. 
 +
 Note that each variable is defined within a `Properties`  object with a `Variable Unique ID`.  The `Variable Unique ID` enables the variable to be tracked within your flow. The `Variable Unique ID` can contain only alphanumeric characters, hyphens, and underscores. When using the Visual Editor, this ID is automatically generated for you.
-Each variable must specify the `Name` and data type. To set the properties for the variable definition, complete the following: +
+Each variable must specify the `Title` and data type. To set the properties for the variable definition, complete the following: +
 
-... `Variable Label (Title)`: The display label for the variable. (Required) 
-... `Type`: The data type of the variable. The following xref:ms_composer_custom_expression_editor.adoc#data-types[data types] are supported: `String`, `Number`, and `Boolean`. To define a `Date` or `DateTime` variable, the data type should be set to `String` and the `Format` property should be defined. (Required) 
-... `Format`:  The qualifier for `String` properties that can specify additional data types not natively supported in JSON. For variables, this value must be set when defining `Date` or `DateTime` variables as `date` or `date-time` respectively. (Optional)
+.. `Variable Label (Title)`: The display label for the variable. (Required) 
+.. `Type`: The data type of the variable. The following xref:ms_composer_custom_expression_editor.adoc#data-types[data types] are supported: `String`, `Number`, `Boolean` and `Array`. To define a `Date` or `DateTime` variable, the data type should be set to `String` and the `Format` property should be defined. (Required) 
+.. `Format`:  The qualifier for `String` properties that can specify additional data types not natively supported in JSON. For variables, this value must be set when defining `Date` or `DateTime` variables as `date` or `date-time` respectively. (Optional)
```

Comment:
I will stop deleting the initial articles in the property descriptions. They are not needed and since we are trying to cut text, it is good to remove them.

Reply:
Fixed throughout